core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -675,6 +685,47 @@ public static boolean defaultUseHttp1Pipelining() {
         return DEFAULT_USE_HTTP1_PIPELINING;
     }
 
+    /**
+     * Returns a boolean indicating whether to send <a href="https://httpwg.org/specs/rfc7540.html#PING">PING</a>
+     * frames on an idle HTTP/2 connection.
+     * If true, then PINGs are send when the connection is idle, no otherwise. Connection is considered idle
+     * when there is no read or write for at least {@link Flags#defaultClientIdleTimeoutMillis()} for client and
+     * {@link Flags#defaultServerIdleTimeoutMillis()} ()} for server.
+     *
+     * <p>This flag is disabled by default. Specify the
+     * {@code -Dcom.linecorp.armeria.useHttp2PingOnIdle=true} JVM option to enable it.
+     */
+    public static boolean useHttp2PingOnIdle() {
+        return USE_HTTP2_PING_ON_IDLE;
+    }
+
+    /**
+     * Returns a boolean indicating whether to <a href="https://httpwg.org/specs/rfc7540.html#PING">PING</a>
+     * frames on a HTTP/2 connection when it is idle and there are no active HTTP/2 stream.
+     * A connection is considered having no active streams means that are no requests and responses pending.
+     * This does not necessarily indicate end of life of a HTTP/2 connection.
+     * \A client could create a new stream if it intends to use the same connection.
```

Comment:
Do we need to document any interaction between this and the other flag?

Reply:
nit: backslash